### PR TITLE
fix: pre bundle webpack-dev-server and webpack loaders

### PR DESCRIPTION
**Pull request by ClarkXia**
新增预打包：
- webpack-dev-server
- work-loader、url-loader、raw-loader
- PWA 相关：workbox-window、workbox-webpack-plugin

相关预打包逻辑 https://github.com/ice-lab/builder-deps/pull/18

### CODE REVIEW

File: packages/build-babel-config/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "url": "https://github.com/alibaba/ice/tree/master/packages/build-babel-config"
   },
   "dependencies": {
-    "@builder/pack": "^0.4.0"
+    "@builder/pack": "0.4.6-2"
```
Comment (SoloJiang) [inline packages/build-babel-config/package.json:21]: 这里后面要改下吧?

**Review by SoloJiang — APPROVED** [review]

File: packages/build-babel-config/package.json (hunk 1 above)
Comment (ClarkXia) [reply packages/build-babel-config/package.json]: 已统一修改版本

File: packages/plugin-pwa/package.json [hunk 2]
```diff
@@ -10,19 +10,17 @@
...
+    "build-scripts": "^1.0.1",
     "typescript": "^4.0.0",
     "webpack-chain": "^6.4.0"
   },
   "dependencies": {
-    "build-scripts": "^1.0.1",
-    "fs-extra": "^9.1.0",
-    "workbox-webpack-plugin": "^6.1.5",
-    "workbox-window": "^6.1.5"
+    "@builder/pack": "0.4.6-2",
```
Comment (luhc228) [inline packages/plugin-pwa/package.json:22]: 0.4.6-2 -> 0.5.0

Comment (ClarkXia) [reply packages/plugin-pwa/package.json:22]: fixed

**Review by luhc228 — APPROVED** [review]

**Pull request merged**